### PR TITLE
feature: create `datashare` CLI on macOS

### PR DESCRIPTION
**Pull request by ClemDoum**
# TODO
- [x] handle forwarding args from the `datashare` CLI to the JAR in a similar way as in the `launchBack.sh` of DS or `entrypoint.sh` of the docker image. (done in #15 )

# Changes

### Added
- create `datashare` CLI by creating a symlink if the command does not exists yet

### CODE REVIEW

**Comment by ClemDoum** [comment]
@mvanzalu how to you test this, I couldn't install build dependency on my machine :s

**Review by pirhoo — APPROVED** [review]

**Review by pirhoo — APPROVED** [review]

File: mac/scripts/postinstall [hunk 1]
```diff
@@ -29,6 +29,18 @@ is_datashare_installed () {
   log_info "Datashare $ds_version is already installed."
 }
 
+create_datashare_cli () {
+    if ! command -v datashare 1>/dev/null 2>&1 ; then
+        sudo ln -s /Applications/Datashare.app/Contents/MacOS/Datashare.command /usr/local/bin/datashare
+        # TODO: uncomment this to exit here in case of failure
+#        if [ $? -ne 0 ]; then
+#            log_error "Unable to create Datashare CLI"
+#            return 1
```
Comment (pirhoo) [inline mac/scripts/postinstall:38]: Maybe you can just comment the return so we keep the log?

**Review by pirhoo — APPROVED** [review]

**Comment by ClemDoum** [comment]
handle forwarding args before merging...

**Comment by pirhoo** [comment]
Forwarding args was handled here https://github.com/ICIJ/datashare-installer/pull/15

**Pull request merged**